src/Markdown.MAML/Renderer/MamlRenderer.cs

```diff
@@ -75,331 +31,161 @@ public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands)
         /// <returns></returns>
         public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands, bool skipPreambula)
```

Comment:
Please remove `skipPreambula`, since you are removing support for it.
It's not used anymore any way. You would need to remove it from the line

```
$xml = $r.MamlModelToString($maml, $false) # skipPreambula is not used
```

in platyPS.psm1 as well